creators/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+
+from creators.views import ProductCreateView, FileUploadView
+
+urlpatterns = [
+    path('/creating', ProductCreateView.as_view()),
```

Comment:
피드백 1. URI는 `POST :8000/product`가 되도록 만들어주세요.
피드백 2. creator app -> product app으로 코드를 옮겨주세요.
(resource 중심으로 디자인 해주세요.)

```suggestion
    path('/product', ProductView.as_view()),
```